libcxx/include/iosfwd

```diff
@@ -25,7 +25,6 @@ template<>            struct char_traits<wchar_t>;
 
 template<class T>     class allocator;
 
-class ios_base;
```

Comment:
This is part of the synopsis, I don't think it should change?

Reply:
I guess it should change to match [[iosfwd.syn]](https://eel.is/c++draft/iosfwd.syn).

Reply:
This is intended and indeed to match the current synopsis. (This is mentioned in the commit message.)